tests/dir.props

```diff
@@ -47,8 +47,40 @@
     <CoreClrPackageVersion>1.0.4-beta-24325-02</CoreClrPackageVersion>
 
     <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(Microsoft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)$</CoreFxVersionsIdentityRegex>
+
+    <BaseDotNetBuildInfoUrl>https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/</BaseDotNetBuildInfoUrl>
+    <DependencyBranch>master</DependencyBranch>
```

Comment:
It would, but I don't see the value in setting it via the build system:

 1. The plan is to use this for dependency validation, so everyone working on the branch needs the same value. (And I don't think there's a reliable way for dev builds to determine "current branch" without checking something into source.)
 2. Even before that goes into action, currently the value is only used to upgrade dependencies, and it isn't used during builds.
 3. The logic below is simply `$(BaseDotNetBuildInfoUrl)[...]/$(DependencyBranch)` in this PR, but it will be customized for different branches. E.g. `dev/api` would still depend on `master` for External, but `dev/api` for corefx and coreclr. (On the corefx side, once coreclr starts building `dev/api`, the corefx `dev/api` branch needs to start using that as its dependency. (If that's actually planned.))

I can't think of an alternative that doesn't require some property checked into source that is different based on branch.